config.js

```diff
@@ -2,9 +2,20 @@ function capitalize(string) {
     return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
 }
 
+function get_live_dos_framework(environment) {
+    if (environment=="Preview") {
+        return "digital-outcomes-and-specialists-5"
+    }
+    else {
+        return "digital-outcomes-and-specialists-4"
+    }
+}
+
 var environment = capitalize(process.env.DM_ENVIRONMENT || 'development')
 var domain = process.env.DM_FRONTEND_DOMAIN || 'http://localhost'
 
+var live_dos_framework = get_live_dos_framework(environment)
```

Comment:
This could be more idiomatic as

```suggestion
var live_dos_framework = environment == (
    "Preview" ? "digital-outcomes-and-specialists-5" : "digital-outcomes-and-specialists-4"
)
```

Although I think I prefer your version 😉

Reply:
I actually wrote it like that first before thinking I was being too clever....